rasa/cli/test.py

```diff
@@ -115,29 +116,51 @@ def run_core_test(args: argparse.Namespace) -> None:
     )
 
 
-def run_nlu_test(args: argparse.Namespace) -> None:
-    """Run NLU tests."""
+async def run_nlu_test_async(
+    all_args: Optional[Dict] = None,
```

Comment:
I'd suggest to make this the last argument (similar as `additional_arguments` in `perform_nlu_cross_validation` or `test_nlu`

Reply:
Moved `all_args` as the last parameter in `run_nlu_test_async`.